docs/apib/squisher.rb

```diff
@@ -1,26 +1,26 @@
 require 'json'
 
 input_file = ARGV.first
-output = "output.apib"
+output = 'output.apib'
 
-def squish(json) 
-  if json.is_a?(Array) and json.size > 0
-    [squish(json.first)]
-  elsif json.is_a?(Hash)
-    json.map { |key, value| [key, squish(value)] }.to_h
-  else
-    json
+def squish(json)
+  if json.kind_of?(Array)
+    return [squish(json.first)] if json.empty?
```

Comment:
There is a semantic change here, the code before didn't enter in this `if` of the json was empty `json.size > 0` and you return a value if it is empty, was it intentional?